container/Dockerfile.gpu

```diff
@@ -71,7 +71,7 @@ RUN apt-get update && \
 COPY --from=builder /workspace/python/dist/*.whl /home/model-server/
 
 RUN pip3 install --pre --no-cache-dir multi-model-server \
-    && pip3 install --no-cache-dir --prefer-binary numpy scipy xlrd Pillow boto3 six requests mxnet-cu100 tensorflow_gpu \
+    && pip3 install --no-cache-dir --prefer-binary numpy scipy xlrd Pillow boto3 six requests mxnet-cu100 tensorflow_gpu opencv-python \
```

Comment:
Did you test the inference container using this ? I think open cv also needs some ubunutu packages.

Reply:
Install the missing ubuntu package and tested inference container in SM